cpp/src/NVStrings.cu

```diff
@@ -345,17 +334,10 @@ int NVStrings::to_host(char** list, int start, int end)
     for( unsigned int idx=0; idx < count; ++idx )
     {
         if( h_strings[idx]==0 )
```

Comment:
`continue` leads to spaghetti logic. It's clearer to write:

```
for( unsigned int idx=0; idx < count; ++idx )
{
  if( h_strings[idx]!=0 ) // Skip null strings
  {
    size_t offset = h_offsets[idx];
    size_t len = h_offsets[idx+1] - offset;
    const char* p_data = h_buffer + offset;
    if( list[idx] )
      memcpy(list[idx], p_data, len-1);
  }
}
```

Reply:
I have to disagree with this. I think using continue for exception cases at the top of a loop is better than adding extra indenting for the main code logic for the loop's purpose.